pkg/apis/alerterator/v1alpha1/alert.yaml

```diff
@@ -62,22 +65,24 @@ spec:
                 - for
                 - expr
                 - action
-              properties:
-                alert:
-                  type: string
-                description:
-                  type: string
-                expr:
-                  type: string
-                for:
-                  type: string
-                  pattern: '\d+[mh]'
-                action:
-                  type: string
-                documentation:
-                  type: string
-                sla:
-                  type: string
-                severity:
-                  type: string
-                  pattern: 'good|warning|danger|#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})'
+              items:
```

Comment:
Hva gjør denne?

Reply:
For å validere en array så må man spesifisere `items`, ellers så vil ikke noen av elementene valideres